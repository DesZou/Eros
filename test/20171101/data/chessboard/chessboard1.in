3
o*o
ooo
*oo
