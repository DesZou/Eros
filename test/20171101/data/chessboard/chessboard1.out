1
119
8
0
